app/controllers/cinema_halls_controller.rb

```diff
@@ -1,55 +1,60 @@
 class CinemaHallsController < ApplicationController
-    before_action :set_cinema_hall, only: [:show, :update, :destroy]
     def index
-        @cinema_halls = CinemaHall.all.map do |cinema_hall|
-            {
-                name: cinema_hall.name,
-                capacity: cinema_hall.capacity
-            }
-        end
-        
-        render json: @cinema_halls
-    end
-
-    def show
-        @cinema_hall = {
-            id: @cinema_hall.id
-            name: @cinema_hall.name,
-            cpacity: @cinema_hall.capacity
-        }
-        render json: @cinema_hall
+        @cinema_halls = UseCases::CinemaHalls::FetchForCinemaHalls.new.call
```

Comment:
Compare the indentation here with any other file in the repository committed before, for example: https://github.com/agusiol/montecinema/blob/main/config/application.rb#L23 . As you can spot your file indentation has 4 spaces, the other one has an indentation of 2 spaces. You should keep it consistent for many reasons, mostly to avoid "noise" in the repository. 

Here you can find how to set it in VS Code: https://stackoverflow.com/questions/34174207/how-to-change-indentation-in-visual-studio-code